nf_core/pipelines/create/create.py

```diff
@@ -112,7 +112,14 @@ def __init__(
         self.template_features_yml = yaml.safe_load(rendered_features)
 
         # Get list of files we're skipping with the supplied skip keys
-        self.skip_paths = set(sp for k in self.skip_areas for sp in self.template_features_yml[k]["skippable_paths"])
+
+        skip_paths: list = []
+        for feature in self.skip_areas:
+            for section in self.template_features_yml.values():
+                if feature in section["features"]:
+                    if section["features"][feature]["skippable_paths"]:
+                        skip_paths = skip_paths + section["features"][feature]["skippable_paths"]
+        self.skip_paths = set(skip_paths)
```

Comment:
```suggestion
skip_paths = [
    path
    for feature in self.skip_areas
    for section in self.template_features_yml.values()
    if feature in section["features"] and section["features"][feature]["skippable_paths"]
    for path in section["features"][feature]["skippable_paths"]
]
self.skip_paths = set(skip_paths)
```
feels more pythony. but if it is more difficult to read for you, keep the easier to read version